environments/autostart.md

```diff
@@ -0,0 +1,51 @@
+---
+title: "Autostart"
+description: Learn how to configure automated environment rebuilds.
+---
+
+Coder automatically [turns off](../admin/environment-management/shutdown.md)
+unused environments to help you save on cloud usage costs. Turning off
+environments, however, requires you to turn them on manually the next day.
+Autostart can help eliminate this manual process.
+
+Autostart also allows you to configure automated environment rebuilds at a time
+that best suits your workflow. You can now expect your environments to be ready
+for you at the start of each workday.
```

Comment:
Currently working on rephrasing and combining these paragraphs. Here's where I am currently:

```md
Coder
[automatically turns off idle environments](../admin/environment-management/shutdown.md)
to help manage resource expenditure. Typically, this means environments turn off
overnight and remain offline until a rebuild is requested. With Autostart,
rebuilds may be automatically requested at a time that best suits your workflow.
You can expect your environments to be ready for you at the start of each
workday.
```